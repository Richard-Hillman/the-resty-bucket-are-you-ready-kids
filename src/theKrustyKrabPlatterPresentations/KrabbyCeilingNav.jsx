import React, { Component } from 'react';


// This page will have the navigation 
export default class KrabbyCeilingNav extends Component {

  render() {
    return (
      <div>

        <div>
          <h1>Welcome to The Resty Krab!</h1>
        </div>

      </div>
    );
  }
}
