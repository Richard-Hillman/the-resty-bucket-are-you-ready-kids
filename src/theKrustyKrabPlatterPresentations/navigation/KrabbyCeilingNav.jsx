import React, { Component } from 'react';
import styles from './KrabbyCeilingNav.css';

// This page will have the navigation 
export default class KrabbyCeilingNav extends Component {

  render() {
    return (
      <div className={styles.KrabbyCeilingNav}>

        <div className="krab2">
          <h1>Welcome to The Resty Krab!</h1>
        </div>

      </div>
    );
  }
}
