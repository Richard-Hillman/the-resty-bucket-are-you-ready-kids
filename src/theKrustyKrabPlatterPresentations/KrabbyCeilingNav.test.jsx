import React from 'react';
import { render, cleanup } from '@testing-library/react';
import KrabbyCeilingNav from './KrabbyCeilingNav';

describe('KrabbyCeilingNav component', () => {
  afterEach(() => cleanup());
  it('renders KrabbyCeilingNav', () => {
    const { asFragment } = render(<KrabbyCeilingNav />);
    expect(asFragment()).toMatchSnapshot();
  });
});
