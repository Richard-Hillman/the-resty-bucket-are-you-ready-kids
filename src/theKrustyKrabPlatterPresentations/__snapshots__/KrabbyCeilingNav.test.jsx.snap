// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KrabbyCeilingNav component renders KrabbyCeilingNav 1`] = `
<DocumentFragment>
  <div>
    <div>
      <h1>
        Welcome to The Resty Krab!
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
